leetcode/README.md

```diff
@@ -0,0 +1,69 @@
+# 📚 Contributing 📚
+
+We're glad you're interested in adding C++ LeetCode solutions to the repository.\
+Here we'll be explaining how to contribute to LeetCode solutions properly.
+
+## 💻 Cloning/setting up the project 💻
+
+First off, you'll need to fork the repository [**here**](https://github.com/TheAlgorithms/C-Plus-Plus/fork).\
+Then, you'll need to clone the repository to your local machine.
+
+```bash
+git clone https://github.com/your-username/C-Plus-Plus.git
+```
+
+After that, you'll need to create a new branch for your solution.
+
+```bash
+git checkout -b solution/your-solution-name
+```
+
+## 📝 Adding a new solution 📝
+
+All LeetCode problems can be found [**here**](https://leetcode.com/problemset/all/).\
+If you have a solution to any of these problems (which are not being **repeated**, that's great! Here are the steps:
+
+1. Add a new file in `leetcode/src` with the number of the problem.
+    - For example: if the problem's number is 98, the filename should be `98.cpp`.
+2. Provide a small description of the solution at the top of the file. A function should go below that. For example:
+
+```c
+/**
+ * Return an vector of vector of size returnSize.
+ * The sizes of the vectors are returned as returnColumnSizes vector.
+ * Note: Both returned vector and columnSizes vector must be dynamically allocated, assume caller calls free.
+ */
+```
+
+3. Do not provide a `main` function. Use the required standalone functions instead.
+4. Doxygen documentation isn't used in LeetCode solutions. Simple/small documentation or comments should be fine.
+5. Don't include libraries/headers such as `iostream`. Your file should be the solution to the problem only.
```

Comment:
This might be needed to be added in the files to prevent CI failures, unless we ignore the files in the CI, which is not 100% recommended, though.
What do you suggest we do here? I suggest using libraries/headers in the files to prevent CI failures.

Reply:
I was thinking whether this directory could be excluded from the CI, as in the case of the  [C](https://github.com/TheAlgorithms/C) repo leetcode directory.  Else we also need to add a driver code for each problem along with some test cases. What are your thoughts?

Reply:
Hi @Panquesito7 
Updated the cpp files and the README file for these changes.